docs/usage.rst

```diff
@@ -60,24 +60,24 @@ You can read the signal:
 
 .. jupyter-execute::
 
-    sig, fs = audiofile.read('noise.flac')
-    print(f'sampling rate: {fs}, signal shape: {sig.shape}')
+    signal, sampling_rate = audiofile.read('noise.flac')
+    print(f'sampling rate: {sampling_rate}, signal shape: {signal.shape}')
 
 If you prefer a workflow
 that returns a 2D signal with channel as the first dimension,
 enforce it with:
 
 .. jupyter-execute::
 
-    sig, fs = audiofile.read('noise.flac', always_2d=True)
-    print(f'sampling rate: {fs}, signal shape: {sig.shape}')
+    signal, sampling_rate = audiofile.read('noise.flac', always_2d=True)
+    print(f'sampling rate: {sampling_rate}, signal shape: {signal.shape}')
 
 If you just want to read from 500 ms to 900 ms of the signal:
 
 .. jupyter-execute::
 
-    sig, fs = audiofile.read('noise.flac', offset=0.5, duration=0.4)
-    print(f'sampling rate: {fs}, signal shape: {sig.shape}')
+    signal, sampling_rate = audiofile.read('noise.flac', offset=0.5, duration=0.4)
+    print(f'sampling rate: {sampling_rate}, signal shape: {signal.shape}')
```

Comment:
Below we use a different output format:

```
    print(f'sampling rate: {audiofile.sampling_rate("noise-remix.flac")}')
    print(f'channels: {audiofile.channels("noise-remix.flac")}')
    print(f'samples: {audiofile.samples("noise-remix.flac")}')
```

I would suggest we align it.

Reply:
I updated it to

![image](https://user-images.githubusercontent.com/173624/217466639-830d2982-0fa9-4456-909b-4606725d85aa.png)

and adjusted the other examples accordingly, see the screenshots in the description.